R/series__series.R

```diff
@@ -61,34 +93,104 @@
 #' s_copy = s
 #' .pr$Series$append_mut(s, pl$Series(5:1))
 #' identical(s_copy$to_r(), s$to_r()) # s_copy was modified when s was modified
+#'
+#' # call functions via sub-namespaces
+#' pl$Series(c(1:3))$expr$add(1)
+#'
+#' pl$Series(list(3:1, 1:2, NULL))$list$first()
+#'
+#' pl$Series(c(1, NA, 2))$str$concat("-")
```

Comment:
This should return a Series, not a DataFrame

Reply:
Thanks for catching this!